.github/scripts/sync-templates.js

```diff
@@ -0,0 +1,37 @@
+const path = require('path')
+const semver = require('semver')
+const fse = require('fs-extra')
+
+const repoRoot = path.join(__dirname, "../..");
+const templateDir = path.join(__dirname, "../../templates");
+const templateDirs = require(path.join(templateDir, "package.json")).templates
+let depPkgs  = [];
+for(let subTempDir of templateDirs){
+    const subTempPath = path.join(templateDir, subTempDir, "package.json")
+    if(fse.existsSync(subTempPath)){
+        depPkgs.push(subTempPath)
+    }
+}
+const pkgDirs = require(path.join(repoRoot, "lerna.json")).packages;
+let templatesDeps = {};
+for(let pkgDir of pkgDirs) {
+    const pkgPath = path.join(repoRoot, pkgDir, "package.json");
+    const pkgName = require(pkgPath).name;
+    const pkgVersion = require(pkgPath).version;
+    templatesDeps[`${pkgName}`] = pkgVersion;
+}
+for(let file of depPkgs) {
+    const pkg_ = fse.readJsonSync(file);
+    const dep_ = pkg_.dependencies;
+    let fileChange = false;
+    for(let [key,value] of Object.entries(templatesDeps)){
+        if(dep_[key] && semver.prerelease(semver.minVersion(dep_[key]))) {
+            dep_[key] = `^${value}`;
```

Comment:
What if `value` is an alpha version? `^3.4.1-alpha.b2569834.0`?

Reply:
value ref from target specific version number of package.json file. 
Such a version number is not possible